src/main/java/com/onarandombox/MultiverseCore/anchor/AnchorManager.java

```diff
@@ -142,7 +142,7 @@ public Set<String> getAnchors(Player p) {
             }
             MVWorld mvWorld = this.plugin.getMVWorldManager().getMVWorld(ancLoc.getWorld());
             if (mvWorld != null) {
-                if (this.plugin.getPlayerActionChecker().hasAccessToWorld(p, mvWorld).isUnsuccessful()) {
+                if (!this.plugin.getPlayerActionChecker().hasAccessToWorld(p, mvWorld).asBoolean()) {
```

Comment:
This isn't really what I meant. Something like `if (!hasAnchorAccessInWorld(player, mvWorld)) {`
```java
private boolean hasAnchorAccessInWorld(Player player, MVWorld mvWorld) {
    return plugin.getPlayerActionChecker().hasAccessToWorld(player, mvWorld).isSuccessful();
}
```
Btw... is this about anchor access in a world cause that's what the logged message says. However, the code doesn't seem to indicate that.

Reply:
ahh